### DISCUSSION TITLE
Show custom popup window with mvvm community toolkit?

### DISCUSSION THREAD

**Discussion by Shadowblitz16** [AvaloniaUI/Avalonia#16210] [Q&A] [answered]
How do I use the mvvm community toolkit to show a custom popup window?
I was reading this https://docs.avaloniaui.net/docs/tutorials/music-store-app/opening-a-dialog but it uses examples for RactiveUI

**Comment by timunie** [comment]
check Avalonia.Samples -> ViewInteraction

**Reply by Shadowblitz16** [reply]
> check Avalonia.Samples -> ViewInteraction

This uses reactive ui also. 
I am looking for one that uses Mvvm toolkit's ObservableObject

**Reply by timunie** [reply, marked as answer]
https://github.com/AvaloniaUI/Avalonia.Samples/tree/main/src%2FAvalonia.Samples%2FViewInteraction%2FDialogManagerSample

That's MVVM Framework independent.  Also the other sample shows own implementation of it if you want to omit RxUI but the DialigManager is my personal favorite.

**Comment by stevemonaco** [comment]
You can try using https://github.com/mysteryx93/HanumanInstitute.MvvmDialogs but this is something that apps typically write themselves when using Mvvm Toolkit though it's not particularly straightforward for beginners. There are a fair number of opinions, plus different app integrations (window vs overlay) that are to be considered.
